dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -546,22 +548,22 @@ public void ds3DeleteObject(final boolean isButtonEvent) {
         if (Guard.isNullOrEmpty(values)) {
             if (root.getValue() == null) {
                 LOG.info("No files selected");
-                Ds3Alert.show(resourceBundle.getString("error"), resourceBundle.getString("noFiles"), Alert.AlertType.ERROR);
+                alert.showAlert(resourceBundle.getString("noFiles"));
             } else {
                 final ImmutableList.Builder<TreeItem<Ds3TreeTableValue>> builder = ImmutableList.builder();
-                values = builder.add(root).build().asList();
+                values = builder.add(root).build().asList();  //DM TODO why?
```

Comment:
Looking closer at it, I think you are right.  Go ahead and remove it and see what happens.